tests/Unit/ApparentHorizons/Test_YlmSpherepack.cpp

```diff
@@ -0,0 +1,664 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+#include <random>
+
+#include "ApparentHorizons/YlmSpherepack.hpp"
+#include "tests/Unit/ApparentHorizons/YlmTestFunctions.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+using SecondDeriv = YlmSpherepack::SecondDeriv;
+
+void test_prolong_restrict() {
```

Comment:
Try to replace the for loops over grid points with a call to `CHECK_ITERABLE_APPROX(a, b)`